### DISCUSSION TITLE
How can I create new plugin?

### DISCUSSION THREAD

**Discussion by Mohan-vidya** [chatwoot/chatwoot#6857] [Q&A]
I want to create a plugin.
Does chatwoot support plugin integration if yes then provide me with an example of a plugin or no How to integrate a new feature without touching chatwoot core code? Please guide me and give me an example of the code if possible.

**Comment by sojan-official** [comment]
The easiest option to handle custom use cases is to use dashboard apps : https://www.chatwoot.com/hc/user-guide/articles/1677691702-how-to-use-dashboard-apps
